k2/csrc/log.h

```diff
@@ -179,11 +188,18 @@ class Logger {
         printf("\n\n%s\n", stack_trace.c_str());
       }
       fflush(nullptr);
+      if (EnableAbort()) {
+      	abort();
+	// raise(SIGTERM);
+      }
+      else {
+     	throw std::runtime_error(kErrMsg);
+      }
       // abort();
       //
       // NOTE: abort() will terminate the program immediately without
       // printing the Python stack backtrace.
-      throw std::runtime_error(kErrMsg);
+      // throw std::runtime_error(kErrMsg);
```

Comment:
```cpp
if (EnableAbort()) {
  abort();
}

throw std::runtime_error(kErrMsg);
```

is simpler.  

From https://en.cppreference.com/w/cpp/utility/program/abort, 
`abort` will never return, so you don't need to a use a separate `else` here.

Reply:
I see.